src/gitea/issue/base.py

```
@@ -0,0 +1,201 @@
+
+        Args:
+            owner: The owner of the repository.
+            repository: The name of the repository.
+
+        Returns:
+            The endpoint URL for listing issues.
+
+        """
+        return f"/repos/{owner}/{repository}/issues"
+
+    def _list_issues_helper(  # noqa: PLR0913
```

Ruff (0.14.13)

Unused `noqa` directive (non-enabled: `PLR0913`)

Remove unused `noqa` directive

(RUF100)